test/battle/battle-order.test.ts

```diff
@@ -24,121 +22,110 @@ describe("Battle order", () => {
 
   beforeEach(() => {
     game = new GameManager(phaserGame);
-    game.override.battleType("single");
-    game.override.enemySpecies(Species.MEWTWO);
-    game.override.enemyAbility(Abilities.INSOMNIA);
-    game.override.ability(Abilities.INSOMNIA);
-    game.override.moveset([MoveId.TACKLE]);
+    game.override
+      .battleType("single")
+      .enemySpecies(Species.MEWTWO)
+      .enemyAbility(Abilities.INSOMNIA)
+      .ability(Abilities.INSOMNIA)
+      .moveset([MoveId.TACKLE]);
   });
 
   it("opponent faster than player 50 vs 150", async () => {
-    await game.startBattle([Species.BULBASAUR]);
+    await game.classicMode.startBattle([Species.BULBASAUR]);
 
     const playerPokemon = game.scene.getPlayerPokemon()!;
     const enemyPokemon = game.scene.getEnemyPokemon()!;
-    vi.spyOn(playerPokemon, "stats", "get").mockReturnValue([20, 20, 20, 20, 20, 50]); // set playerPokemon's speed to 50
-    vi.spyOn(enemyPokemon, "stats", "get").mockReturnValue([20, 20, 20, 20, 20, 150]); // set enemyPokemon's speed to 150
+    vi.spyOn(playerPokemon, "stats", "get").mockReturnValue([100, 20, 20, 20, 20, 50]); // set playerPokemon's speed to 50
+    vi.spyOn(enemyPokemon, "stats", "get").mockReturnValue([100, 20, 20, 20, 20, 150]); // set enemyPokemon's speed to 150
+    game.scene.getField(true).forEach((p) => (p.hp = p.getMaxHp()));
 
     game.move.select(MoveId.TACKLE);
-    await game.phaseInterceptor.run(EnemyCommandPhase);
+    await game.phaseInterceptor.to("BerryPhase", false);
```

Comment:
```suggestion
    await game.toEndOfTurn();
```
Make sure to change all the other ones you added in other files too.

Reply:
I just did a find-and-replace... sorry for bloating the file changes 😬 

![image](https://github.com/user-attachments/assets/2912dd0c-7dc3-4ca8-abcd-2d81acbc228c)

Reply:
That's fine, it needed to be done at some point anyway lol.